androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/CheckEligibilityTask.java

```diff
@@ -55,8 +80,9 @@ protected void buildParams() {
         addOptionalParam("language[code]", settings.getLanguageCode());
         addOptionalParam("language[product_name]", settings.getProductName());
         addOptionalParam("language[audience_text]", settings.getRecommendTarget());
+        addOptionalParam("end_user_last_seen", preferencesUtils.getLastSeen() / 1000);
```

Comment:
No, `getLastSeen` is of type `long` so the result will be of this type, no floating point.